### PR TITLE
Documents

### PR DESCRIPTION
**Pull request by kyle-manganyi**

### CODE REVIEW

File: skietbaan_fe/src/components/ViewComp.js [hunk 1]
```diff
@@ -1,417 +1,522 @@
...
+        },
+        body: JSON.stringify(this.state.numberofshots)
+      }
+    )
+      .then(function(response) {})
+      .then(function(data) {})
+      .catch(function(data) {});
+  }
+
+  MakeLetterOfStatus(CompID) {
```
Comment (ilzeLourens) [inline skietbaan_fe/src/components/ViewComp.js]: camelCase

File: skietbaan_fe/src/components/ViewComp.js [hunk 2]
```diff
@@ -1,417 +1,522 @@
...
+  changeStatus(status, i) {
+    const newCompOBJArr = [...this.props.compOBJ]; //cloning an array of competition
+    newCompOBJArr[i].status = !status; //changing the status of a newly cloned array
+    this.props.updateByIdComp(newCompOBJArr[i], i + 1); //updating the competition status by ID in the back-end
+    const newDict = { ...this.props.dict };
+    this.setState({ compOBJ: newCompOBJArr });
+    this.setState({ dict: newDict });
+  }
+  //the method that adds the update for the inputs
+  HandleOnSubmit = index => {
```
Comment (JurieOberholzer) [inline skietbaan_fe/src/components/ViewComp.js]: camelCase

File: skietbaan_fe/src/components/ViewComp.js [hunk 3]
```diff
@@ -1,417 +1,522 @@
...
+    this.updateNumberOfShots;
+    this.MakeLetterOfStatus;
```
Comment (JurieOberholzer) [inline skietbaan_fe/src/components/ViewComp.js]: same here

File: skietbaan_fe/src/components/ViewComp.js [hunk 4]
```diff
@@ -1,417 +1,522 @@
...
+      accuracy: this.state.silverAccuracy,
+      total: this.state.silverTotal
+    };
+    const GoldData = {
+      standard: "Gold",
+      accuracy: this.state.goldAccuracy,
+      total: this.state.goldTotal
+    };
+    const RData = [BronzeData, SilverData, GoldData];
+    this.props.UpdateRequirements(index, RData);
```
Comment (JurieOberholzer) [inline skietbaan_fe/src/components/ViewComp.js]: same here

File: skietbaan_fe/src/components/ViewGroups.js [hunk 5]
```diff
@@ -90,7 +90,7 @@ class ViewGroups extends Component {
                 <tr className="view-group" key={post.id}>
                   <td
                     className={
-                      post.isActive === true ? "first-row" : "first-row-active"
+                       post.isActive === true ? "first-row" : "first-row-active"
```
Comment (JurieOberholzer) [inline skietbaan_fe/src/components/ViewGroups.js]: remove space

File: skietbaan_fe/src/components/ViewComp.js [hunk 6]
```diff
@@ -1,417 +1,522 @@
...
-		this.setState({ [target.name]: target.value });
-	}
+  changeToggle() {
+    this.setState({ toggleRequirements: !this.state.toggleRequirements });
+  }
 
-	changeToggle() {
-		this.setState({ toggleRequirements: !this.state.toggleRequirements });
-	}
+  changeStatusOfLOGS() {
```
Comment (JurieOberholzer) [inline skietbaan_fe/src/components/ViewComp.js]: change to  changeStatusOfLogs

File: skietbaan_fe/src/actions/types.js [hunk 7]
```diff
@@ -45,7 +45,7 @@ export const UPDATE_SELECTED_COMPETITION = "UPDATE_SELECTED_COMPETITION";
 export const UPDATE_SELECTED_GROUP = "UPDATE_SELECTED_GROUP";
 
 //groups types and constants
-export const URL = "http://localhost:50860";
```
Comment (Sibusiso-Retro) [inline skietbaan_fe/src/actions/types.js]: already have base URL on line 39, please remove this one

File: skietbaan_fe/src/actions/competition.action.js [hunk 8]
```diff
@@ -1,99 +1,116 @@
...
-					dispatch({
-						type: NEW_COMP,
-						payload: true
-					});
-				});
-			}
-		});
-};
\ No newline at end of file
+export const createcomp = compData => dispatch => {
```
Comment (JurieOberholzer) [inline skietbaan_fe/src/actions/competition.action.js]: camelCase

**Pull request merged**